rest/route/service.go

```diff
@@ -102,9 +102,9 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	app.AddRoute("/hosts/{host_id}/change_password").Version(2).Post().Wrap(checkUser).RouteHandler(makeHostChangePassword(sc, env))
 	app.AddRoute("/hosts/{host_id}/extend_expiration").Version(2).Post().Wrap(checkUser).RouteHandler(makeExtendHostExpiration(sc))
 	app.AddRoute("/hosts/{host_id}/terminate").Version(2).Post().Wrap(checkUser).RouteHandler(makeTerminateHostRoute(sc))
-	app.AddRoute("/hosts/{host_id}/status").Version(2).Get().Wrap(checkUser).RouteHandler(makeContainerStatusManager(sc))
-	app.AddRoute("/hosts/{host_id}/logs/output").Version(2).Get().Wrap(checkUser).RouteHandler(makeContainerLogsRouteManager(sc, false))
-	app.AddRoute("/hosts/{host_id}/logs/error").Version(2).Get().Wrap(checkUser).RouteHandler(makeContainerLogsRouteManager(sc, true))
+	app.AddRoute("/hosts/{host_id}/status").Version(2).Get().RouteHandler(makeContainerStatusManager(sc))
```

Comment:
We need some sort of authentication on these routes. You can maybe reuse the host check that some of the agent routes use

Reply:
Adding something, although I won't be able to test until I get staging access.